adagrams/game.py

```diff
@@ -1,11 +1,96 @@
+import random
+
+
+LETTER_POOL = ['A','A','A','A','A','A','A','A','A','B','B','C','C','D',
+        'D','D','D','E','E','E','E','E','E','E','E','E','E','E','E','F',
+        'F','G','G','G','H','H','I','I','I','I','I','I','I','I','I','J',
+        'K','L','L','L','L','M','M','N','N','N','N','N','N','O','O','O',
+        'O','O','O','O','O','P','P','Q','R','R','R','R','R','R','S','S',
+        'S','S','T','T','T','T','T','T','U','U','U','U','V','V','W','W',
+        'X','Y','Y','Z']
+
+
 def draw_letters():
```

Comment:
Hi Gloria! Great job completing Adagrams! Congrats on your first project at Ada 🎉 Excellent choice with this implementation